### ISSUE TITLE
Delay JavaScript Execution compatibility with Customer Reviews for WooCommerce

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4555]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.10.4`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
When [Customer Reviews for WooCommerce](https://wordpress.org/plugins/customer-reviews-woocommerce/) is used with Delay JavaScript Execution, and `/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js` was previously excluded, the reviews grid doesn't render on the page, and the following dependency error shows up on the browser's console: 

![image](https://user-images.githubusercontent.com/1289174/144407162-fa0b34a3-816d-42cb-aeaa-85b9a3d7825d.png)

```
Uncaught TypeError: jQuery(...).colcade is not a function
    at HTMLDivElement.<anonymous> (frontend.js:48)
    at Function.each (jquery.min.js:2)
    at s.fn.init.each (jquery.min.js:2)
    at crResizeAllGridItems (frontend.js:46)
    at HTMLDocument.<anonymous> (frontend.js:1)
    at e (jquery.min.js:2)
    at t (jquery.min.js:2)
```

Excluding the following fixes the issue: 

```
/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js 
/jquery-migrate(.min)?.js
/customer-reviews-woocommerce/js/colcade.js
```

**To Reproduce**
1. Use Delay JavaScript Execution with the Customer Reviews for WooCommerce
2. Add a reviews grid on the page
3. See error

**Expected behavior**
Customer Reviews for WooCommerce should work out of the box when Delay JavaScript Execution is enabled.

**Screenshots**
Added on the description part.

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1694288221/307317/
Added to Delay JavaScript Execution exclusions document - https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions#customer-reviews-for-woocommerce

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by piotrbak** [comment]
@DahmaniAdame I wasn't able to reproduce this error. I believe that some previous exclusions caused this issue to happen.

**Comment by DahmaniAdame** [comment]
@piotrbak the issue only happens when jquery was previously excluded :)
Updated the main post to include that detail 👍

**Comment by piotrbak** [comment]
Closed by 3.13 backend
